src/scss/hpe/_hpe.defaults.scss

```diff
@@ -46,7 +47,7 @@ $brand-graph-colors: (#FF8D6D, #877B75, #2AD2C9, #614767, #617D78);
 
 html,
 .brand-font {
```

Comment:
Hmm, would that mean all font selectors for all themes would be in the _base.font.scss?  Trying to make sure I understand the suggestion.